package.json

```diff
@@ -31,6 +31,6 @@
     "sinon": "^1.17.7"
   },
   "peerDependencies": {
-    "structured-log": "^0.1.0 || ^0.0.18"
+    "structured-log": "^0.2.0" || "^0.1.0 || ^0.0.18"
```

Comment:
`"^0.2.0" || "^0.1.0 || ^0.0.18"` should be `"^0.2.0 || ^0.1.0 || ^0.0.18"` (note the extra quotes) - wanna fix, then I'll merge in the PR?